dist/config.yaml

```diff
@@ -0,0 +1,135 @@
+version: 2
```

Comment:
this directory is generated by running goreleaser, we shouldn't commit this to the repository